app/controllers/media_controller.rb

```diff
@@ -29,21 +29,25 @@ def stream
 
   private
 
+  def allowed_params
+    params.permit(:action, :id, :file_name, :format)
```

Comment:
This is interesting, is `action` necessary to add here?  I don't recall needing to be explicit about things like `action` and `controller` before, but I could be totally wrong.